app/util/data_preparation/confluence/prepare-data.py

```diff
@@ -90,6 +90,13 @@ def write_test_data_to_files(dataset):
     __write_to_file(CONFLUENCE_USERS, users)
 
 
+def __check_default_confluence_language(confluence_api):
+    confluence_language = confluence_api.get_locale()
+    if confluence_language != 'en_US':
+        raise SystemExit(f'Unfortunately, we do not support {confluence_language} language! '
```

Comment:
`f'{confluence_language} language is not supported. Please change your profile language **to** English'`

Reply:
done